crypto_data.py

```diff
@@ -1,13 +1,15 @@
 import os
-import requests
+import httpx
```

Comment:
where's your pip requirements file to reflect this change?

Reply:
Will add in next commit, really should've had it but thought since this is a service i'm running myself I didn't think I'd need it